test/trento/software_updates_test.exs

```diff
@@ -183,4 +183,37 @@ defmodule Trento.SoftwareUpdates.SettingsTest do
 
     assert {:error, :settings_already_configured} = SoftwareUpdates.save_settings(settings)
   end
+
+  describe "clearing software update settings" do
+    test "should clear settings" do
```

Comment:
Would you say that this test is a subset of the other test `should support idempotent sequential clear settings`?

Reply:
Yes, we could squash in the second test only. Deal?

Reply:
Sounds like a deal :handshake: